roles/edpm_fips/meta/argument_specs.yml

```diff
@@ -0,0 +1,42 @@
+---
+argument_specs:
+  # ./roles/edpm_fips/tasks/main.yml entry point
+  main:
+    short_description: The main entry point for the edpm_fips role.
+    options:
+      edpm_fips_mode:
+        type: str
+        default: check
+        description: >
+          Specify the action for the role. Valid values are: check,
+          enable, disable.
+      edpm_fips_packages:
+        type: list
+        default:
+          - crypto-policies-scripts
+        description: List of packages required by the role.
+      edpm_fips_grub_params_file:
+        type: path
+        default: /etc/default/grub
+        description: Path of the grub parameter file
+      edpm_fips_grub_config_file:
+        type: path
+        default: /boot/grub2/grub.cfg
+        description: Path of the grub config files
+      edpm_fips_fms_status:
+        type: list
+        default:
+          - {exit_code: 0, message: 'FIPS is enabled'}
+          - {exit_code: 1, message: 'FIPS setup is inconsistent'}
+          - {exit_code: 2, message: 'FIPS is disabled'}
+        description: >
+          Map of fips-mode-setup exit code and message as specified
+          in the command man page.
+      edpm_fips_iscsid_chap_algorithms:
+        type: str
+        default: 'SHA3-256,SHA256,SHA1'
```

Comment:
Can we leave out SHA1?  Or are there devices that still require this?

Reply:
This is a requirement I guess. But i let @Akrog replying to this. :)

Reply:
I think the safe bet is to go with whatever we had in OSP17 to avoid regressions, unless someone can confirm that there are no customers with Storage Arrays that require it.

For reference, I believe there are customers with Storage Arrays that only support MD5 and therefore cannot use FIPS mode in OSP.